ores/wsgi/routes/scores.py

```diff
@@ -113,19 +115,38 @@ def score_revision(context, model, rev_id):
             return responses.not_found("No models available for {0}"
                                        .format(context))
 
-        precache = "precache" in request.args
-
-        # If the model exists, score revisions with it and return the result
         if model not in score_processor[context]:
             return responses.not_found("Model '{0}' not available for {1}."
                                        .format(model, context))
-        else:
-            try:
-                scores = score_processor.score(context, model, [rev_id],
-                                               precache=precache)
-            except errors.ScoreProcessorOverloaded:
-                return responses.server_overloaded()
 
-            return jsonify(scores)
+        caches = {}
+        # Parse values for features / datasources of interest.
+        try:
+            for k, v in request.values.items():
+                if k == "caches":
+                    caches = json.loads(v)
+                    break
+                elif k.startswith(("feature.", "datasource.")):
+                    depth = caches
+                    path = k.split(".")
+                    last = len(path) - 1
+                    for i, key in enumerate(path):
```

Comment:
The keys are flat, right? Like `caches["feature.123.456"]`. So this depth spelunking isn't really desired?